### DISCUSSION TITLE
🧩 FEATURE REQUEST: Weekend mode

### DISCUSSION THREAD

**Discussion by sk2andy** [dailydotdev/daily#1007] [Feature Requests]
## Feature Request

Please add a weekend/off day mode that not reset your weekly goals.

### Description

For some devs reaching the weekly goals is not possible due to weekends. For example I am a family father of two children and cannot read any news article on the weekend or on vacation.

### Solution

Transform the weekly goals to a more generic goal system where you can specify your off days.

### Alternate Solutions

* Do not completely reset the weekly article read count but decrease it by one every day you have not read any article. So devs with no time at the weekend can make at least 3 points a week.
* Give every dev a deposit of like 100 off days in a year

**Comment by idoshamun** [comment]
Hey @sk2andy,

Thanks for raising this concern.
Work-life balance is very important and we will consider this issue in our upcoming product meeting.
I'll keep you posted :)

**Comment by rebelchris** [comment]
This has since been implemented
